src/ui/Logo/Logo.jsx

```diff
@@ -5,41 +5,21 @@
 
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { defineMessages, useIntl } from 'react-intl';
 import { Image } from 'semantic-ui-react';
-import { useSelector } from 'react-redux';
-import config from '@plone/volto/registry';
-
-import LogoImage from '@eeacms/volto-eea-design-system/../theme/themes/eea/assets/images/Header/eea-logo.svg';
-
-const messages = defineMessages({
-  site: {
-    id: 'Site',
-    defaultMessage: 'Site',
-  },
-  eeasite: {
-    id: 'European Environment Agency',
-    defaultMessage: 'European Environment Agency',
-  },
-});
 
 /**
  * Logo component class.
  * @function Logo
  * @param {Object} intl Intl object
  * @returns {string} Markup of the component.
  */
-const Logo = ({ id }) => {
-  const lang = useSelector((state) => state.intl.locale);
-  const intl = useIntl();
-  const url = config.settings.isMultilingual ? `/${lang}` : '/';
-
+const Logo = ({ src, id, url, alt, title }) => {
```

Comment:
@tiberiuichim Can we use defaults here and have `src`, `alt` and `title` fallback to EEA WWW logo?

Reply:
The defaults are possible, but would be useless and would promote bad habits. The actual url, as you see, is dependent on the current language and the messages require translation infrastructure. I think it's better if we keep this clean and simple.